extensions/rich_text_components/Image/directives/ImageDirective.js

```diff
@@ -71,18 +71,14 @@ oppia.directive('oppiaNoninteractiveImage', [
           };
           $scope.loadImage();
         } else {
+          $scope.isLoadingIndicatorShown = false;
+          $scope.isTryAgainShown = false;
```

Comment:
Since these two are set before the if else, we don't need to set them again. (same for ImageClickInput.js)

Reply:
Done!